src/main/webapp/app/quiz/participate/quiz-scoring-infostudent-modal/quiz-scoring-infostudent-modal.component.html

```diff
@@ -0,0 +1,37 @@
+<ng-template #scoringexplanation let-c="close" let-d="dismiss">
+    <div class="modal-header">
+        <h5 class="modal-title">Scoring type explanation</h5>
+        <button type="button" class="close" aria-label="Close" (click)="d()"><span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <div class="modal-body" *ngIf="this.scoringType === 'ALL_OR_NOTHING' && this.questionType === 'mcq'" >
+        <h6>The scoring type was <span jhiTranslate="arTeMiSApp.quizExercise.scoringType.all_or_nothing"></span> This means you either get all the points for the question or non at all.</h6>
+        <span *ngIf="this.wrongAnswers == 0"> You have achieved {{this.score}} of the {{this.question}} possible points. Since you have {{this.wrongAnswers}}
+            wrong answers you have achieved all points in this exercise.</span>
+        <span *ngIf="!(this.wrongAnswers == 0)"> You have achieved {{this.score}} of the {{this.question}} possible points. Since you made {{this.wrongAnswers}} mistakes when selecting the answer options you have achieved 0 points in this exercise.</span>
+    </div>
+    <div class="modal-body" *ngIf="this.scoringType === 'ALL_OR_NOTHING' && this.questionType === 'dnd'" >
+        <h6>The scoring type was <span jhiTranslate="arTeMiSApp.quizExercise.scoringType.all_or_nothing"></span> This means you either get all the points for the question or non at all.</h6>
+        <span *ngIf="this.wrongAnswers == 0"> You have achieved {{this.score}} of the {{this.question}} possible points. Since you have mapped all the items correctly you have achieved all the point. Great Job! </span>
+        <span *ngIf="!(this.wrongAnswers == 0)"> You have achieved {{this.score}} of the {{this.question}} possible points. Since you made {{this.wrongAnswers}} mistakes when dragging the elements into the correct places you have achieved 0 points.</span>
+    </div>
+    <div class="modal-body" *ngIf="this.scoringType === 'PROPORTIONAL_WITH_PENALTY' && this.questionType === 'mcq' ">
+        <h6>The scoring type was <span jhiTranslate="arTeMiSApp.quizExercise.scoringType.proportional_with_penalty"></span> Meaning each correct answer results in a fraction of the total score. To avoid guesswork, the same fraction is subtracted from the achieved points for each mistake.</h6>
```

Comment:
please use the translation system to provide this text in English and German using translateValues for multiple params